src/main/java/io/split/android/client/storage/db/SplitDao.java

```diff
@@ -0,0 +1,20 @@
+package io.split.android.client.storage.db;
+
+import androidx.lifecycle.LiveData;
+
+import androidx.room.Dao;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+import androidx.room.Update;
+
+import java.util.List;
+
+@Dao
+public interface SplitDao {
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
```

Comment:
we will need remove operation for split.status !== ACTIVE

Reply:
@mmelograno , good catch! Will add it.